src/client/client.ts

```diff
@@ -150,29 +150,29 @@ export class ClusterClient {
   }
 
   private checkExpiry() {
-    if(this.isTokenExpired() && this.tokenExpiryHandler) {
```

Comment:
Q: What's the reason to pull this?

Reply:
Currently we have two clients, discovery and native kubernetes one, both are using same logic for token expiry handler. Actually, just realizing that from now on they both will always use the oauth token, as we are working with single `host` cluster. Will adjust the logic a bit, thanks for noting.